example/dotnet/src/Program.cs

```diff
@@ -0,0 +1,2 @@
+﻿// See https://aka.ms/new-console-template for more information
+Console.WriteLine("Hello, World!");
```

Comment:
Fine for this PR, but we could also consider adding this project to the integration tests so that way we know the extension works with this version of dotnet.
https://github.com/ryanluker/vscode-coverage-gutters/blob/master/test/extension.test.ts#L183-L204